### ISSUE TITLE
Cart Explorer: Cart Explorer reminder is not spoken if Cart Insert mode is active

### ISSUE THREAD

**Issue by josephsl** [josephsl/stationPlaylist#17]
Hi,

A long-standing bug (oversight) found: if Cart Insert mode is active and once Cart Explorer is on, NVDA will not announce that one is in this mode when turning Cart Edit off.

STR:

1. From Studio window, press Control+Insert twice to enter Cart Insert mode.
2. Press Alt+NVDA+3 to enter Cart Explorer.
3. Press Control+T to toggle Cart Edit on and then off.

Expected: NVDA announces one is in Cart Explorer.
Actual: No announcement.
Cause: only Cart Edit flag is covered.

Thanks.
